plugins/techdocs/src/reader/components/TechDocsPageHeader.tsx

```diff
@@ -19,12 +19,13 @@ import { AsyncState } from 'react-use/lib/useAsync';
 import CodeIcon from '@material-ui/icons/Code';
 import { EntityName } from '@backstage/catalog-model';
 import { Header, HeaderLabel, Link } from '@backstage/core';
+import { TechDocsMetadata } from '@backstage/techdocs-common';
```

Comment:
The `@backstage/techdocs-common` is only usable in the backend and techdocs-cli. :)
This means we need to duplicate the type - one for `plugins/techdocs` and one for `plugins/techdocs-backend` and `packages/techdocs-common`.